K2Bridge.Tests.UnitTests/JsonConverters/AggregationConverterTests.cs

```diff
@@ -90,6 +90,43 @@ public class AggregationConverterTests
                 }
             }}";
 
+        private const string PercentileAggregation = @"
+            {""aggs"": { 
+                ""2"": {
+                    ""percentiles"": {
+                        ""field"": ""metric"",
+                        ""percents"": [
+                            50
+                        ]
+                    }
+                }
+            }}";
+
+        private const string PercentilesAggregation = @"
+            {""aggs"": { 
+                ""2"": {
+                    ""percentiles"": {
+                        ""field"": ""metric"",
+                        ""percents"": [
+                            1, 50, 90, 95
+                        ]
+                    }
+                }
+            }}";
+
+        private const string PercentilesKeyedAggregation = @"
```

Comment:
whats the difference to the one who is not keyed?

Reply:
Median: keyed = true
Percentiles: keyed = false